src/interfaces/IDrips.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.8.0;
+
+interface IDrips {
```

Comment:
Was this interface generated from `cast` or did we get it from the Radicle source?

Reply:
Hand built it from reading Radicle source code.. now using interface generated by `cast interface` command.
Done: e0ed98a23eb6ba0d25c48873324f027b1d48cb97

Reply:
Done: c0ebe7abf0a1a325645990082e24790f671c2159